packages/client/src/__tests__/dmmf.test.ts

```diff
@@ -324,7 +326,55 @@ describe('dmmf', () => {
       }
     `)
   })
+})
+
+describeIf(process.env.PRISMA_CLI_QUERY_ENGINE_TYPE === 'library')('dmmf library', () => {
```

Comment:
Maybe instead of duplicating here and skipping for binary in `packages/client/src/__tests__/generation/generator.test.ts`

We could try to normalize / search and replace `query-engine-node-api library` to something like `query-engine-NORMALIZED` or so
With a if 
```
if (process.env.PRISMA_CLI_QUERY_ENGINE_TYPE === 'binary') {
 expect(stripAnsi(e.message).toContain("query-engine binary"))
} else {
expect(stripAnsi(e.message).toContain("query-engine-node-api library"))
}
```

What do you think?